Wootric/WootricSDK-Android
# Add showSurveyInActivity method

Add new method to let users specify in which activity
they wish to show the survey

## Changes

- Add showSurveyInActivity method
- Add new test

## How to test

1. Setup an Android project with two different activities.
2. Setup Wootric in the first activity
3. Go to the other activity (with an `Intent` or however you chose to do that) and then call Wootric from that activity with

```java
wootric.showSurveyInActivity(SecondActivity.this);
```

It doesn't matter if you setup wootric in the first activity with that context, it should now show the survey in the specified activity.

## Trello

https://trello.com/c/ctoPOjkD/4295-android-sdk-add-showsurveyinactivity-method
